### PR TITLE
ebuild-maintenance/package-moves: add common terms for pkgmove/slotmove

### PR DESCRIPTION
**Pull request by thesamesam**

### CODE REVIEW

File: ebuild-maintenance/package-moves/text.xml [hunk 1]
```diff
@@ -5,9 +5,9 @@
 <body>
 
 <p>
-This chapter describes the use of package and slot moves. The package update
-mechanism is a powerful tool, and needs to be used with caution. In particular,
-the following must be noted:
+This chapter describes the use of package ("pkgmove" / "move") and slot moves
+("slotmove"). The package update mechanism is a powerful tool, and needs to be
+used with caution. In particular, the following must be noted:
```
Comment (ulm) [inline ebuild-maintenance/package-moves/text.xml:10]: Not sure if these technical terms should be mentioned already in the introductory sentence of the chapter. They're explained in their proper context below.

**Review by ulm — COMMENTED** [review]
Rest looks good.

File: ebuild-maintenance/package-moves/text.xml (hunk 1 above)
Comment (thesamesam) [reply ebuild-maintenance/package-moves/text.xml:10]: I take the point on _slotmove_, but wanted to include _pkgmove_ at least as it's an unofficial term which is often used on IRC and would help people search for it. But then figured if I was including one, I should mention the other 😄. What do you think?

File: ebuild-maintenance/package-moves/text.xml [hunk 2]
```diff
@@ -77,6 +78,9 @@ and update them. These include:
   <li>
     open bug summaries
   </li>
+  <li>
+    wiki articles
```
Comment (ulm) [inline ebuild-maintenance/package-moves/text.xml:82]: IIUC the technical term is "wiki pages".

**Review by ulm — APPROVED** [review]
One small suggestion, otherwise LGTM.

File: ebuild-maintenance/package-moves/text.xml (hunk 2 above)
Comment (thesamesam) [reply ebuild-maintenance/package-moves/text.xml]: Thank you!

**Pull request closed without merging**